assets/app.js

```diff
@@ -647,8 +595,14 @@ new Vue({
             else{
                 var response = []
                 this.rawMaterials.forEach(function (option, index) {
-                    if(option.name !== '-' && option.is_active==1){
-                        response.push(option);
+                    if(IsAdmin == true){
```

Comment:
you should extract this to a separate function, you're repeating yourself :slightly_smiling_face: